scalahost/nsc/src/main/scala/scala/meta/internal/semantic/DenotationOps.scala

```diff
@@ -92,7 +92,7 @@ trait DenotationOps { self: DatabaseOps =>
     }
 
     def toDenotation: m.Denotation = {
-      m.Denotation(flags, name, info)
+      m.Denotation(flags, name, info, gsym.pos.toMeta)
```

Comment:
Judging by the output of https://travis-ci.org/scalameta/scalameta/jobs/257894597, `Symbol.pos` is not quite what we need. Looks like in the Scala compiler, this position is the range of the entire definition, whereas in Scalameta we want only the range of the name of the definition.